tests/integration/models/setup_utils.py

```diff
@@ -661,3 +658,46 @@ def compare_values(
         # Add a small absolute tolerance to avoid problems with zero values.
         rtol = 1e-5 * np.sum(np.abs(values_0))
         assert np.isclose(np.sum(values_0 - values_1), 0, atol=1e-10 + rtol)
+
+
+def get_testable_methods_names(model_setup) -> list[str]:
+    """Get all possible testable methods for the test_eval_methods_... tests.
+
+    Parameters:
+        model_setup: Model setup after `prepare_for_simulation()`.
+
+    Returns:
+        List of all possible testable method names for the given model.
+
+    """
+
+    # Get all public methods
+    all_methods = [method for method in dir(model_setup) if not method.startswith("_")]
+
+    # Get all testable methods
+    # A testable method is one that:
```

Comment:
Perhaps move this definition to function docstring?

Reply:
Will do.